### PR TITLE
fix: fixed bad env variable for TOKEN_NAME && feat: Fetch dynamically ETH price

### PR DESCRIPTION
**Pull request by 0xJord4n**

### CODE REVIEW

**Comment by 0xJord4n** [comment]
fb please

**Review by enitrat — COMMENTED** [review]
Thanks for the PR ! 👯

File: src/index.ts [hunk 1]
```diff
@@ -195,9 +196,10 @@ async function main() {
   } else {
     processedBlock = newprocessedBlock;
   }
+  ethPrice = await getETHPriceInUSD();
   await inspectBlockBuys(latestBlock, BigInt(TOKEN_ADDRESS));
 }
 
-cron.schedule("*/30 * * * * *", main);
+cron.schedule("*/1 * * * * *", main);
```
Comment (enitrat) [inline src/index.ts]: Not a great idea imo as blocktime is something around 30secs, so anything lower than 10s is completely overkill imo

Comment (0xJord4n) [reply src/index.ts]: So, let's use 10s because with 30 we can miss txs

**Comment by 0xJord4n** [comment]
Just edited the time

**Pull request merged**